ollama.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: ollama
+  version: 0.0.16
+  epoch: 0
+  description: Get up and running with Llama 2 and other large language models locally
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - build-base
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/jmorganca/ollama
+      tag: v${{package.version}}
+      expected-commit: 177b69a211755537dc9bb8fe548ba3f596e8e3c5
+
+  - runs: |
+      CGO_ENABLED=1 go build -ldflags '-linkmode external -extldflags "-static"' .
```

Comment:
I assume we don't use `go/build` here because we want CGO?  Maybe we should change `go/build` to allow cgo to be enabled?

Reply:
Yeah it can do ldflags but not env vars (that I know of)

Reply:
or extldflags